pycheribuild/projects/cross/juliet_test_suite.py

```diff
@@ -66,13 +68,20 @@ class BuildJulietCWESubdir(CrossCompileCMakeProject):
     do_not_add_to_targets = True
     cwe_number = None
     default_install_dir = DefaultInstallDir.DO_NOT_INSTALL
+    cwe_warning_flags = []
```

Comment:
If you add this before adding the CWEs you won' have a commit with contents that are immediately removed.

Reply:
Swapped the commits